### ISSUE TITLE
Scale is not reflected in the segment drawing.

### ISSUE THREAD

**Issue by komikoni** [bbc/peaks.js#290]
I change the scale to 2 because the recorded waveform information is small.
However, segment information is only drawn with half the waveform.

```
                        const instance = Peaks.init(options, function (err, peaks) {
                            // Do something when the waveform is displayed and ready
                            const zoomview = peaks.views.getView('zoomview');
                            zoomview.setAmplitudeScale(2.0);
                            const overview = peaks.views.getView('overview');
                            overview.setAmplitudeScale(2.0);
                        });
```
![image](https://user-images.githubusercontent.com/3490654/68544992-3398ec00-040c-11ea-8b0d-f59e288d9a7a.png)

I forcibly
With "WaveformZoomView"
"WaveformOverview"
Rewritten to call setAmplitudeScale of the segment layer.
Overview was successful,
ZoomView failed. (Success at first)

I think ZoomView is rewritten frequently.

```
  WaveformZoomView.prototype.setAmplitudeScale = function(scale) {
    if (!Utils.isNumber(scale) || !Number.isFinite(scale)) {
       throw new Error('view.setAmplitudeScale(): Scale must be a valid number');
    }

    this._waveformShape.setAmplitudeScale(scale);
    this._waveformLayer.draw();
    this._segmentsLayer.setAmplitudeScale(scale); //add 

  };
```
![image](https://user-images.githubusercontent.com/3490654/68545044-8ffc0b80-040c-11ea-8a90-07b31974b814.png)

Next, I rewrote the fixed setting `this._scale = 1.0;` in the WaveformShape function of waveform-shape.js.

```
  function WaveformShape(options) {
    Konva.Shape.call(this, {
      fill: options.color
    });

    this._view = options.view;
    this._segment = options.segment;
    // this._scale = 1.0;
    this._scale = 2.0;

    this.sceneFunc(this._sceneFunc);
  }

```

![image](https://user-images.githubusercontent.com/3490654/68545124-6a233680-040d-11ea-9d68-f261151eaec4.png)

Finally it was as expected.

Is this a bug? Or is there a way to set the scale when drawing the segment?

**Comment by chrisn** [comment]
Yes, this is a bug. I will push a fix soon. Sorry for the inconvenience.

**Comment by chrisn** [comment]
I have just published release 0.14.5 that fixes this issue. Thank you for the feedback!
